### DISCUSSION TITLE
Pre-trained CNN's for optical flow estimations

### DISCUSSION THREAD

**Discussion by davemadams** [community/community#120236] [Programming Help]
I'm looking for pre-trained optical flow CNNs to implement in Python. So far, l've only found RAFT, which I implemented using PyTorch. Are there any other architectures that are easy to implement in Python? I'd be grateful for any help
I tried it with GitHub clones but most of the time they do not work properly. I now hope for CNNs to implement via Keras or something like that?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @davemadams !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Code Search and Navigation category is solely related to conversations around the GitHub product CS&N. This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
